pkg/sensors/program/program.go

```diff
@@ -84,6 +84,10 @@ type Program struct {
 
 	MultiSymbols []string
 	MultiCookies []uint64
+
+	// uprobe setup
+	Path   string
+	Symbol string
```

Comment:
Can we use `LoaderData` for these fields?

We can have a:
```
type UprobeData struct {
      Path string
      Symbol string
}
```

set it in `SetUprobe` and in load time do a run-time cast to get it.

Reply:
right, will check on that

Reply:
hum, so we already use LoaderData for passing data to LoadProbe interface
and it needs to contain generic uprobe data like `ID` to get uprobeEntry

while the UprobeAttach function is in generic program loader code which just
needs path and symbol for uprobes.. not necessarily generic in future ;-) 

I don't like mixing the genericuprobe sensor data with data for generic loader

maybe we could add another `AttachData` data holder to Program ,  like:

```
        // LoaderData represents per-type specific fields.
        LoaderData interface{}
 
+       // AttachData represents specific data for attaching probe
+       AttachData interface{}
+
        MapLoad []*MapLoad
```

and we would use it for uprobe specific attach data there and also data for multi kprobe data
which are now placed in Program's:

```
        MultiSymbols []string
        MultiCookies []uint64
```

Reply:
Makes sense to me, thanks!